apps/examples/testcase/le_tc/kernel/tc_environ.c

```diff
@@ -87,6 +87,15 @@ static void tc_environ_setenv_getenv_unsetenv(void)
 	psz_getvalue = getenv("arv");
 	TC_ASSERT_EQ("getenv", psz_getvalue, NULL);
 
+	ret_chk = setenv(NULL, psz_value, overwrite_num);
+	TC_ASSERT_EQ("setenv", ret_chk, ERROR);
+
+	ret_chk = setenv(psz_name, NULL, overwrite_num);
+	TC_ASSERT_EQ("setenv", ret_chk, OK);
+
+	ret_chk = setenv(psz_name, NULL, 0);
```

Comment:
Why don't you use fixed value instead of overwrite_num variable in here? If there is no benefit of using variable, let's remove it. (I think there is no benefit.)